cmd.js

```diff
@@ -0,0 +1,67 @@
+const Hyperbeam = require('./')
+
+if (global.Bare) process = require('bare-process') // eslint-disable-line
```

Comment:
You can also `require('process')` and then add an `imports` map to `package.json` that remaps `process` to `bare-process` in Bare.